catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -586,7 +624,6 @@ extension MDCNodeListViewController {
         cell!.detailTextLabel?.text = subtitleText!
       }
       cell!.accessibilityIdentifier = "Cell" + cell!.textLabel!.text!
-      cell!.accessoryType = .disclosureIndicator
```

Comment:
I think the disclosure indicator may have been left out as an oversight - on iOS it's a common affordance to indicate that drill-down navigation is possible and I'd like to ensure that we keep that affordance. Can we add the disclosure indicators back?